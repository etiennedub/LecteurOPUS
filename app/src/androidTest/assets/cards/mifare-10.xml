<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">044F25E6</List><List index="1">0A674980</List><List index="2">A448E000</List><List index="3">00000002</List><List index="4">00010001</List><List index="5">3129C520</List><List index="6">6F540000</List><List index="7">9ED65A1D</List><List index="8">00000000</List><List index="9">00000000</List><List index="10">000138D4</List><List index="11">00006948</List><List index="12">01C2C532</List><List index="13">5333D50A</List><List index="14">4100B8D4</List><List index="15">20280462</List></Pages></Mifare>