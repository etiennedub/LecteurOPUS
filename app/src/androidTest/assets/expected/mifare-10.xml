<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="30/09/2016" Time="11:05" Transfert="false" OperatorId="20" BusId="317" OperatorName="RTC" BusName="METROBUS 807" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="1" OperatorId="20" OperatorName="RTC" /></Contract></Card>
